src/hosted-buttons/index.js

```diff
@@ -19,6 +23,7 @@ export const getHostedButtonsComponent = (): HostedButtonsComponent => {
   function HostedButtons({
     enableDPoP = false,
     hostedButtonId,
+    fundingSources = [],
```

Comment:
After our latest meeting with NCP team, they will provide a new variable `version` in our function call `getHostedButtonDetails`. I left this feature flag in for now since that is what was outlined in the ticket when I started.

I can update it to match what we will expect from the above response or we can update it once we actually have the updated response.